README.rst

```diff
@@ -69,7 +69,7 @@ Somewhere in your server's request handler code:
            format=Format.HTTP_HEADERS,
            carrier=request.headers,
        )
-       span = tracer.start_span(
+       span = tracer.start_manual_span(
```

Comment:
here we have `tracer.start_active_span` and `tracer.start_manual_span`. Any other API is deprecated and then will be removed; mostly following the Java API, though we don't have the `_span` suffix but found that in the original PR: https://github.com/opentracing/opentracing-python/pull/50/files#r118851809